cts/README

```diff
@@ -72,121 +89,75 @@ A complete command line might look like:
+  if you have some other type of key (RSA, ECDSA, etc.), use its file name
+  in the -i option above.
 
-	cat /home/myid/.ssh/identity.pub >> ~root/.ssh/authorized_keys
+  If you have an old version of SSH that doesn't have ssh-copy-id,
+  you can take the single line out of your public key file
+  (e.g. ~/.ssh/identity.pub or ~/.ssh/id_dsa.pub) and manually add it to
+  root's ~/.ssh/authorized_keys file on each test machine.
 
-	To test this, try this command from the exerciser machine for each
-	of your cluster machines, and for the exerciser machine itself.
+* To test, try this command from the exerciser machine for each
+  of your cluster machines, and for the exerciser machine itself.
 
-ssh -l root cluster-machine
+	ssh -l root $MACHINE
 
-If this works without prompting for a password, you're in business...
-If not, you need to look at the ssh/openssh documentation and the output from
-the -v options above...
+  If this works without prompting for a password, you're in business.
+  If not, look at the documentation for your version of ssh.
 
 --------------------------------------------------------------------------------
 How to configure OpenSSH for StonithdTest
 --------------------------------------------------------------------------------
 
-This configure enables cluster machines to ssh over to each other without a
-password challenge.
-
-1)	On each of the cluster machines, grab your key:
-
-	take the single line out of ~/.ssh/identity.pub
-	and put it into root's authorized_keys file.
-	[This has changed to: copying the line from ~/.ssh/id_dsa.pub into
-		root's authorized_keys file ]
-
-	NOTE: If you don't have an id_dsa.pub file, create it by running:
-
-		ssh-keygen -t dsa
-
-2)	Run this command on each of the cluster machines as root:
-
-ssh -v -l myid cluster_machine_1 cat /home/myid/.ssh/identity.pub \
-	>> ~root/.ssh/authorized_keys
-
-ssh -v -l myid cluster_machine_2 cat /home/myid/.ssh/identity.pub \
-	>> ~root/.ssh/authorized_keys
-
-......
-
-ssh -v -l myid cluster_machine_n cat /home/myid/.ssh/identity.pub \
-	>> ~root/.ssh/authorized_keys
-
-[For most people, this has changed to:
-  ssh -v -l myid cluster_machine cat /home/myid/.ssh/id_dsa.pub \
-	>> ~root/.ssh/authorized_keys
-]
-
-	You will probably have to provide your password, and possibly say
-	"yes" to some questions about accepting the identity of the
-	test machines
-
-To test this, try this command from any machine for each
-of other cluster machines, and for the machine itself.
-
-    ssh -l root cluster-machine
-
-This should work without prompting for a password,
-If not, you need to look at the ssh/openssh documentation and the output from
-the -v options above...
+* Follow the above mini-HOWTO for allowing you to login as root
+  on each test machine without providing a password.
 
-3)  Make sure the 'at' daemon is enabled on the test cluster machines 
+* Make sure the 'at' daemon is installed, started and enabled to start at boot
```

Comment:
Removing the obsolete section for now. We can add more STONITH examples in a later revision if desired.